content/en/dashboards/widgets/profiling_flame_graph.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Profiling Flame Graph Widget
+kind: documentation
+description: "Graph a breakdown of top consuming lines of code (CPU, Memory, ...)"
+widget_type: "flame_graph"
+aliases:
+- /graphing/widgets/profiling_flame_graph/
+further_reading:
+- link: "/dashboards/graphing_json/"
+  tag: "Documentation"
+  text: "Building Dashboards using JSON"
+---
+
+The [profiling flame graph visualization][1] represents a breakdown of top consuming lines of code (CPU, Memory, ...):
+
+{{< img src="dashboards/widgets/profiling_flame_graph/profiling_flame_graph.png" alt="Profiling Flame Graph" >}}
+
+## Setup
+
+### Configuration
+
+1. Select the scope for profiling data, such as `host`, `container_name`, `service`, `env` or `version`.
+2. Select the resource, such as `CPU Time`, `Allocated Memory` or `Thrown Exceptions`.
+3. Optional: chose the frame granularity and coloring (`Line`, `Method`, `Class`, `File`, `Package`, ...).
```

Comment:
> If these aren't specified, will the query apply default values. If this is the case, can we reword this:
Yes it will have a default value: `Method` or `Function`.

> Should customers know what these options are or is this worth defining?
I believe most customers are not aware of them. But they may not be worth defining 🤷